addons/account/__manifest__.py

```diff
@@ -82,7 +82,6 @@
         'demo/account_demo.xml',
     ],
     'installable': True,
-    'application': True,
```

Comment:
We still want an application in Community
* `account` is an application (both visible from SaaS app selector and from the "App" app)
* `account_accountant` is not an app, but it is auto-installed
* `accountant` is an application

Reply:
ok